frontend/packages/pipelines-plugin/src/components/repository/__tests__/RepositoryLinkList.spec.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { ResourceLink, ExternalLink, CopyToClipboard } from '@console/internal/components/utils';
+import { PipeLineRunWithRepoMetadata } from '../../../test-data/pipeline-data';
+import RepositoryLinkList from '../RepositoryLinkList';
+
+jest.mock('react-i18next', () => {
+  const reactI18next = require.requireActual('react-i18next');
+  return {
+    ...reactI18next,
+    useTranslation: () => ({ t: (key: string) => key }),
+  };
+});
```

Comment:
This should be already [globally mocked](https://github.com/openshift/console/blob/master/frontend/__mocks__/react-i18next.js) for you. If you _need_ this to avoid errors than our global mock is in error and that's something we should look into.

Reply:
updated